app/src/main/java/app/revanced/integrations/settingsmenu/ReturnYouTubeDislikeSettingsFragment.java

```diff
@@ -62,8 +62,6 @@ public void onCreate(Bundle savedInstanceState) {
         setPreferenceScreen(preferenceScreen);
 
         enabledPreference = new SwitchPreference(context);
-        enabledPreference.setKey(SettingsEnum.RYD_ENABLED.getPath());
-        enabledPreference.setDefaultValue(SettingsEnum.RYD_ENABLED.getDefaultValue());
```

Comment:
Why was this removed?

Reply:
because it's does nothing, and the code gives the impression it does something.